Packs/PasswordResetViaChatbot/README.md

```diff
@@ -0,0 +1,22 @@
+
+Remember that time when couldn't login to your system because you forgot your password? Ever needed the help of IT just after they left for lunch?
+Does a Slack channel full of messages asking to reset passwords sound familiar?
+
+Well, this pack provides you with the perfect solution - resetting passwords automatically using a chatbot.
+Resetting passwords through chatbots and automation can provide numerous benefits to organizations. 
+First, it can reduce the workload on IT departments by allowing users to reset their passwords quickly and easily without needing assistance. This can improve efficiency and reduce costs. 
+Additionally, using chatbots and automation for password resets can enhance the security of the organization by allowing for multifactor authentication and real-time monitoring of login attempts. 
+Furthermore, it can improve the user experience by providing a simple and intuitive interface for resetting passwords, which can help to reduce frustration and improve productivity. 
+By automating password resets, organizations can also ensure that password policies are consistently enforced, reducing the risk of security breaches. 
+
+##### What does this pack do?
+1. Ingests an incident created by a user message on Slack or Microsoft Teams to reset their password.
+2. Seeks approval for password resets from the relevant stakeholder.
+3. Verifies the user using 2-factor authentication.
+4. Creates a new password for the user, meeting the password complexity criteria of the organization.
+5. Sets the new password for the user, requiring the user will to change their password on next login.
```

Comment:
```suggestion
5. Sets the new password for the user, requiring the user will have to change their password on next login.
```

Reply:
I will change this to "...requiring the user to change..."